warehouse/query-core/src/main/java/datawave/query/tracking/ActiveQueryLog.java

```diff
@@ -73,9 +73,9 @@ synchronized public void setLogPeriod(long logPeriod) {
             if (logPeriod != this.logPeriod || this.timer == null) {
                 if (this.timer != null) {
                     this.timer.cancel();
-                    this.timer = new Timer("ActiveQueryLog");
                 }
                 this.logPeriod = logPeriod;
+                this.timer = new Timer("ActiveQueryLog");
```

Comment:
this.timer should be set with a new Timer for either condition - logPeriod != this.logPeriod || this.timer == null
It's the cancel that shouldn't be executed when this.timer is null to avoid an NPE